### ISSUE TITLE
Fresh fork of master cannot run tests

### ISSUE THREAD

**Issue by michaelKaefer** [symfony/symfony#38337]
**Symfony version(s) affected**: 3.4 and master branch

**Description**  
Following the docs to setup Symfony for a PR (https://symfony.com/doc/current/contributing/code/tests.html) produces an error when running the tests with `php ./phpunit symfony`:

```
PHP Fatal error:  Uncaught ReflectionException: Class SymfonyExcludeListSimplePhpunit does not exist in /home/michael/symfony/symfony/.phpunit/phpunit-9.3-0/phpunit:12
Stack trace:
#0 /home/michael/symfony/symfony/.phpunit/phpunit-9.3-0/phpunit(12): ReflectionClass->__construct()
#1 {main}
  thrown in /home/michael/symfony/symfony/.phpunit/phpunit-9.3-0/phpunit on line 12
KO src/Symfony/Bundle/FrameworkBundle
```

**How to reproduce**
Create a new fork, then:
```bash
git clone git@github.com:michaelKaefer/symfony.git
cd symfony
composer update
php ./phpunit symfony
```
Also with:
```bash
git checkout -t origin/3.4
php ./phpunit symfony
```

**Additional context**  
Tried with the following PHP versions (all produce the same error):
- PHP 7.2.33
- PHP 7.3.22
- PHP 7.4.3
- PHP 7.4.10

I use Ubuntu:
```
Linux thinkpad 5.4.0-48-generic #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

I found this issue: https://github.com/symfony/symfony/issues/32995 but that does not help me to successfully run the tests.

**Comment by Yondz** [comment]
Hi there,

Same issue for me. I did some digging, it could be related to #37513, I managed to "fix" it (tests are running) on my side by adding the following piece of code in .phpunit/phpunit-9.3-0/phpunit : 

```php
if (!class_exists('SymfonyExcludeListSimplePhpunit', false)) {
    class SymfonyExcludeListSimplePhpunit {}
}
```

I'm not sure it's the way to go to fix this issue as I am not familiar with the project itself yet, but it could be done by adding those lines at https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php#L270 I guess, can anyone double-check this ?

See ya

**Comment by nicolas-grekas** [comment]
Will be fixed by #38444

**Comment by Yondz** [comment]
Still having the issue on master, I think we have to fix it on > 4.4 as well, since simple_phpunit was splitted into simple_phpunit & simple_phpunit.php (https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php#L261), WDYT @nicolas-grekas ?

**Comment by nicolas-grekas** [comment]
We periodically merge older branches into newest. The patch will come to master this way soon.
